src/qw/remote_repo/_github.py

```diff
@@ -46,16 +50,83 @@ def labels(self) -> list[str]:
     @property
     def body(self) -> str:
         r"""Get the body of the first comment, always using `\n` as the newline character."""
+        if self._issue.body is None:
+            return ""
         return "\n".join(self._issue.body.splitlines())
 
     @property
     def item_type(self) -> RemoteItemType:
-        """Get the type of the issue, as we may handle a pull request differently to an issue."""
-        if self._issue.as_dict().get("pull_request"):
-            return RemoteItemType.REQUEST
+        """Get the type of the issue."""
         return RemoteItemType.ISSUE
 
 
+class PullRequest(qw.remote_repo.service.PullRequest):
+    """A pull request on GitHub."""
+
+    def __init__(
+        self,
+        body,
+        labels,
+        number,
+        title,
+        **kwargs,
+    ) -> None:
+        """
+        Initialize the pull request.
+
+        Use service.get_pull_request(number) instead.
+        """
+        try:
+            self._body = text_under_heading(body, "Description")
+        except QwError:
+            self._body = body
+        self._labels = labels
+        self._number = number
+        self._title = title
+        self._closing_issues = kwargs.get("closingIssuesReferences", [])
```

Comment:
Not strongly held but would it be easier to make it clear that we're not using the other kwargs by doing this?
```suggestion
    def __init__(
        self,
        body,
        labels,
        number,
        title,
        closingIssuesReferences = None,
        **kwargs,
    ) -> None:
        """
        Initialize the pull request.

        Use service.get_pull_request(number) instead.
        """
        try:
            self._body = text_under_heading(body, "Description")
        except QwError:
            self._body = body
        self._labels = labels
        self._number = number
        self._title = title
        self._closing_issues = closingIssuesReferences or []
```